templates/.VERSION

```diff
@@ -1 +1 @@
-0.9.99.0915
\ No newline at end of file
+v0.9.0+dev
```

Comment:
It should probably be `v0.9.99<something>`, so to be `> v0.9.9', which is the closest tag to the `master` branch